### PR TITLE
Weekly digest

### PR DESCRIPTION
**Pull request by dshevtsov**
## This PR is a:

News update

## Summary

When this pull request is merged, it will add generated updates for the passed week to the What's New on Devdocs page.

### CODE REVIEW

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,53 @@ description: |
...
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: Major update
+  date: March 18, 2019
+  link: https://github.com/magento/devdocs/pull/3932
+- description: Updated the [Cloud Architecture comparison chart](https://devdocs.magento.com/guides/v2.3/cloud/architecture/cloud-architecture.html).
+  versions: 2.x
+  type: Technical changes
+  date: March 15, 2019
+  link: https://github.com/magento/devdocs/pull/3893
+- description: Updated the [Fastly troubleshooting](https://devdocs.magento.com/guides/v2.3/cloud/cdn/trouble-fastly.html
```
Comment (lorikrell) [inline _data/whats-new.yml]:
```suggestion
- description: Updated [Fastly troubleshooting](https://devdocs.magento.com/guides/v2.3/cloud/cdn/trouble-fastly.html
```

File: _data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,53 @@ description: |
...
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: Major update
+  date: March 15, 2019
+  link: https://github.com/magento/devdocs/pull/3699
+- description: Updated definition of [Exceptions 5.17](https://devdocs.magento.com/guides/v2.2/coding-standards/technical-guidelines.html#5-exceptions).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: March 15, 2019
+  link: https://github.com/magento/devdocs/pull/3871
+- description: Updated the [Fastly troubleshooting](https://devdocs.magento.com/guides/v2.3/cloud/cdn/trouble-fastly.html
```
Comment (lorikrell) [inline _data/whats-new.yml]:
```suggestion
- description: Updated [Fastly troubleshooting](https://devdocs.magento.com/guides/v2.3/cloud/cdn/trouble-fastly.html
```

**Comment by dshevtsov** [comment]
Thanks @lorikrell 
The changes have to be made at the corresponding PRs.
And seems like it's the same PR. Someone added two labels to indicate type of PR.

**Comment by dshevtsov** [comment]
running tests

**Pull request merged**